File: week-03/day-02/practice/week-02/day-02/draw-pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was
number = int(input())

for i in range(number+1):
    if i % 2 == 1:
        print(" " * (number-i) + "*"*i*2)

File: week-07/day-02/draw-pyramid.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is

var star = '*';
var whitespace = ' ';
for (var i = 1; i <= lineCount*2; i+=2) {
    console.log(whitespace.repeat(lineCount*2-i/2) + star.repeat(i));
}
